packages/frontend/src/components/create-campaign/preview/index.tsx

```diff
@@ -99,6 +103,49 @@ export function CampaignPreview({
         },
     });
 
+    useEffect(() => {
+        const uploadSpecification = async () => {
+            setUploadingSpecification(true);
+
+            const specification: FullSpecification = {
+                // TODO: add restrictions
+                kpi: payload.kpiSpecification,
+            };
+
+            // FIXME: remove
+            console.log(JSON.stringify(specification));
```

Comment:
I believe this could now be removed no?

Reply:
Fixed in 0edb2c51d297f15fb54f2bfbba97283afc4c78e6.